topics/unity/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: unity3d
+created_by: Unity Technologies
+display_name: Unity
+github_url: https://github.com/Unity-Technologies
+logo: unity.png
+short_description: Unity is a game development platform used to build high-quality 3D, and 2D games, deploy them across mobile, desktop, VR/AR, consoles, or the Web.
```

Comment:
The comma before "and 2D" seems unnecessary.

Can you rephrase the short_description to be different from the description? Maybe rework the full description below to have more details than this short_description.

Reply:
I agree - I made that change to make the Oxford command test go from 🔴  to 💚 :

```
  1) Failure:
topics::unity topic#test_0014_follows the Topic Page Style Guide [/Users/leereilly/github/explore/test/topics_test.rb:207]:
Always use the Oxford comma.
Expected "Unity is a game development platform used to build high-quality 3D and 2D games, deploy them across mobile, desktop, VR/AR, consoles, or the Web" to include ", and".
```